### PR TITLE
Release Note for CNV-2692

### PR DESCRIPTION
**Pull request by bgaydosrh**
Adding release note to describe dashboard and limitations. 

See **New and Changed Features > Web Console Improvements**, first bullet.

Please label for Enterprise 4.2.

Peer Review requested @kalexand-rh @mburke5678 @huffmanca 

Tagging @rawagner @jelkosz @aburdenthehand for additional review if they have time.

Thanks!
Bob

### CODE REVIEW

**Comment by bgaydosrh** [comment]
Tagging @phoracek for code review.

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -24,6 +24,8 @@ method. It is not supported for non-default networks.
 * For secondary networks, use the `bridge` binding method.
 
 === Web console improvements
+* The {product-title} dashboard captures high-level information about clusters.
+From the OpenShift web console, access the dashboard by clicking *Home > Dashboards > Overview*. Note that virtual machines are no longer listed in the web console project overview. Virtual machines are now listed within the *Cluster Inventory* card of the dashboard.
```
Comment (kalexand-rh) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/OpenShift/{product-title}
Each UI element must be [enclosed in its own set of asterisks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

**Review by kalexand-rh — COMMENTED** [review]
Two changes and it LGTM.

**Comment by bgaydosrh** [comment]
As @ousleyp is adding this to her Release Note PR, I am closing this story on recommendation from @aburdenthehand.

Final text of release note has been mailed to Pan.

Bob

**Pull request closed without merging**